app/views/about/accessibility.en.html.erb

```diff
@@ -4,6 +4,41 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Accessibility statement</h2>
+    <h1 class="govuk-heading-xl">Accessibility statement</h1>
+
+    <p class="govuk-body">This service is run by the Legal Aid Agency. We want as many people as possible to be able to use this website. For example, that means you should be able to: </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>change colours, contrast levels and fonts</li>
+      <li>zoom in up to 300% without the text spilling off the screen</li>
+      <li>navigate most of the website using just a keyboard</li>
+      <li>navigate most of the website using speech recognition software</li>
+      <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+
+    <p class="govuk-body">We’ve also made the website text as simple as possible to understand.</p>
+
+    <p class="govuk-body"><a aria-label="AbilityNet" href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+
+    <h2 class="govuk-heading-m">How accessible this website is</h2>
+    <p class="govuk-body">This website is fully compliant with levels A, AA and AAA of the Web Content Accessibility Guidelines – WCAG 2.1.</p>
+
+    <h2 class="govuk-heading-m">Reporting accessibility problems with this website</h2>
+    <p class="govuk-body">We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact us by email at <%= mail_to Settings.support_email, class: 'govuk-link--no-visited-state', 'aria-label': Settings.support_email %>.</p>
+
+    <h2 class="govuk-heading-m">Enforcement procedure</h2>
+    <p class="govuk-body">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).</p>
+    <p class="govuk-body">If you’re not happy with how we respond to your complaint, <a aria-label="contact the Equality Advisory and Support Service (EASS)" href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS)</a>.</p>
```

Comment:
Why the `aria-label` in all links?
I think is not necessary when the link copy is the same we want a screen reader to read.